### ISSUE TITLE
ADA Aria/tab index compliance

### ISSUE THREAD

**Issue by luckyth13teen** [metafizzy/flickity#1326]
Hey team, 

Was having a bit of issues with Flickity for ada compliance. Specifically " ARIA hidden element must not be focusable or contain focusable elements
Ensure aria-hidden elements are not focusable nor contain focusable elements"

This is on (packaged) v2.2.1 so this might have been fixed on a more recent version but situation required to resolve at "as is" so this might be edge case functionality. 

As a fix I built out a js listener and modifier to catch and modify these status/declares to meet wcag needs. 

The addon I made is located here: https://github.com/luckyth13teen/Flickity_ada_assist  

Since this was built for specific resolution there is likely some code that is unhelpful/unnecessary, but would rather your team be the judge of its value (or not).
